client/maps/maps-reducer.ts

```diff
@@ -120,6 +120,30 @@ export default immerKeyedReducer(DEFAULT_STATE, {
     }
   },
 
+  ['@matchmaking/getMatchmakingMapPoolsHistory'](state, action) {
```

Comment:
imo, when we're making pages that don't use redux to store their data, it makes more sense to just make (or use) a more general action for loading this data into reducers. Those actions are basically just "here's the data" (and I think one might already exist for maps, I can't recall)